### PR TITLE
refactor: gohan elasticsearch jvm heap options file

### PR DESCRIPTION
**Pull request by v-rocheleau**
Replaced the Elasticsearch JVM heap size environment variable with the preffered configuration file approach.

https://www.elastic.co/guide/en/elasticsearch/reference/current/advanced-configuration.html#set-jvm-heap-size

### CODE REVIEW

File: docs/installation.md [hunk 1]
```diff
@@ -138,6 +138,44 @@ cp ./etc/katsu.config.example.json ./lib/katsu/config.json
...
+
+#### Production
+
+When deploying a Bento node that needs to serve variants data, it is recommended to alot larger portions of your 
+compute resources to both `bentov2-gohan-api` and `bentov2-gohan-elasticsearch`.
+Since variants are stored in memory in Elasticsearch, set the `BENTOV2_GOHAN_ES_MEM_LIM` variable to an 
+appropriate value.
+
+
+Aloting additional CPUs with `BENTOV2_GOHAN_API_CPUS` and `BENTOV2_GOHAN_ES_CPUS` will result in shorter ingestion
```
Comment (davidlougheed) [inline docs/installation.md:151]: Allotting

File: docs/installation.md [hunk 2]
```diff
@@ -138,6 +138,44 @@ cp ./etc/katsu.config.example.json ./lib/katsu/config.json
 ```
 
 
+### `gohan` configuration
+
+#### Production
+
+When deploying a Bento node that needs to serve variants data, it is recommended to alot larger portions of your
```
Comment (davidlougheed) [inline docs/installation.md:145]: allot

**Review by davidlougheed — CHANGES_REQUESTED** [review]
small typo fixes

File: docs/installation.md (hunk 2 above)
Comment (v-rocheleau) [reply docs/installation.md]: oops, thanks "allot"

File: docs/installation.md [hunk 3]
```diff
@@ -138,6 +138,44 @@ cp ./etc/katsu.config.example.json ./lib/katsu/config.json
...
+The heap size can be modified by providing a config file to the `bentov2_gohan-elasticsearch` container:
+
+```bash
+cp ./etc/default.gohan.es.jvm.options ./lib/gohan/es_jvm_options/jvm.options
+```
+
+Set `Xms` and `Xmx` to no more than 50% of the `BENTOV2_GOHAN_ES_MEM_LIM` value.
+For more details, check the official Elasticsearch doc on 
+[heap size](https://www.elastic.co/guide/en/elasticsearch/reference/current/advanced-configuration.html#set-jvm-heap-size).
+
```
Comment (davidlougheed) [inline docs/installation.md:178]: put more newlines here (whatever is needed to match other spaces between headings)

File: .gitignore [hunk 4]
```diff
@@ -47,6 +47,10 @@ lib/public/*
 !lib/public/.gitkeep
 !lib/public/docker-compose.public.yaml
 
+# gohan_elasticsearch
+lib/gohan/es_jvm_options/*
+!lib/gohan/es_jvm_options/.gitkeep
```
Comment (davidlougheed) [inline .gitignore:52]: what happens when the user runs gohan without anything in here; does it work, or should we have a default editable file here?

**Review by davidlougheed — CHANGES_REQUESTED** [review]
two more things, otherwise looks good to me

File: .gitignore (hunk 4 above)
Comment (v-rocheleau) [reply .gitignore:52]: It works with nothing in it as well. This dir binds to `/usr/share/elasticsearch/config/jvm.options.d/`, which is always empty by default . Only files with the `.options` suffix are read by ES there, so whatever is in .gitkeep is ignored.

https://www.elastic.co/guide/en/elasticsearch/reference/current/advanced-configuration.html#set-jvm-options

**Review by davidlougheed — APPROVED** [review]
lgtm

**Pull request merged**